### PR TITLE
View Upcoming Events 

### PR DESCRIPTION
**Pull request by Dragos-U**
Added two reviewers as per Alex request.

### CODE REVIEW

File: src/main/java/ro/itschool/ema/exceptions/GlobalExceptionHandler.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package ro.itschool.ema.exceptions;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
```
Comment (ghost) [inline src/main/java/ro/itschool/ema/exceptions/GlobalExceptionHandler.java:7]: I see a problem here which is that I have already made a controller advice class in my branch (ExceptionHandlerAdvice). How should I proceed?

File: src/main/java/ro/itschool/ema/controllers/EventController.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package ro.itschool.ema.controllers;
+
```
Comment (ghost) [inline src/main/java/ro/itschool/ema/controllers/EventController.java:2]: I have added the Participant Event Registration Endpoint to the StaffController. Should I add it to this class instead?

**Review by ghost — COMMENTED** [review]
Overall, i do not see anything wrong with the code, however, I think there will be some merging issues if this branch or my branch gets merged.

File: src/main/java/ro/itschool/ema/exceptions/GlobalExceptionHandler.java (hunk 1 above)
Comment (Dragos-U) [reply src/main/java/ro/itschool/ema/exceptions/GlobalExceptionHandler.java:7]: they have different names so we can push the code and then have a meeting and refactor the code to keep the best of both sides

File: src/main/java/ro/itschool/ema/controllers/EventController.java (hunk 2 above)
Comment (Dragos-U) [reply src/main/java/ro/itschool/ema/controllers/EventController.java:2]: no, the participant registration will be done from the ParticipantController.

**Pull request merged**